src/index.js

```diff
@@ -0,0 +1,183 @@
+/* Copyright 2012 Mozilla Foundation
```

Comment:
Please note that adding this file really isn't a good idea, since the *public API* is specified in `src/pdf.js` and having a file that you need to *manually* keep in sync isn't a desirable solution from a maintainability perspective.

Furthermore, this is also exporting *a lot* of things that the API doesn't/shouldn't expose.

Reply:
Thanks - that answers my question from before ;)

There is at least one dependency that is not in `pdf.js`, which is `PDFDocumentLoadingTask` - is it OK to add them in the `pdf.js` then?

Reply:
added there for the moment.